spec/std/json/serializable_spec.cr

```diff
@@ -1045,6 +1051,11 @@ describe "JSON mapping" do
     end
   end
 
+  it "parses 128-bit integer" do
+    json = JSONAttrWithInt128.from_json(%({"value": 186709961001538790100634132976990}))
+    json.value.should eq 186709961001538790100634132976990_i128
```

Comment:
1.2.2 CI seems to be failing on this due to using an `Int128` literal if that matters.